ui/js/component/publishForm/view.jsx

```diff
@@ -318,7 +318,7 @@ class PublishForm extends React.PureComponent {
   handleFeePrefChange(feeEnabled) {
     this.setState({
       isFee: feeEnabled,
-      feeAmount: this.state.feeAmount == "" ? "5.00" : this.state.feeAmount,
+      feeAmount: this.state.feeAmount == "" ? "0.01" : this.state.feeAmount,
```

Comment:
It makes sense to define `0.01` as a global constant, assuming it would also be the placeholder / step value for the tipping field.

Reply:
I don't think these are the same value. I would agree to constant this if it was being repeated (I love constants) but right now this is the only place. I did make it a variable to make it clearer.